roles/process_nmstate/defaults/main.yml

```diff
@@ -0,0 +1 @@
+target_name: "{{ inventory_hostname }}"
```

Comment:
With `{{ inventory_hostname }}` var assignments like these, I think it'd be a great practice to leave one or two sentences explaining why that's required for each scenario. This comment could make it a little easier for the reader to understand the execution flow.